### PR TITLE
Document navigator.sendBeacon(...)

### PR DESCRIPTION
**Pull request by jasnell**
workerd PR: https://github.com/cloudflare/workerd/pull/1265

### CODE REVIEW

**Review by irvinebroque — APPROVED** [review]
supportive of shipping under a compat flag to get a sense of usage before enabling by default

**Comment by jasnell** [comment]
We already have a compat flag that controls the availability of the `navigator` global. I wouldn't imagine we'd need another flag specific to this.

**Comment by irvinebroque** [comment]
Oh I misread docs change.

https://developers.cloudflare.com/workers/configuration/compatibility-dates/#global-navigator

This is already enabled by default after 2022-03, so I didn't expect that we'd call out needing to enable it in API docs. I think if _if_ we're going to mention it, we should call out the enabled by default date. Otherwise it comes across like you need to enable it in order to use this API.

**Comment by irvinebroque** [comment]
We should probably have a docs pattern for various APIs to denote the compatibility date at which they were enabled by default, and link to the relevant compatibility flag. (longer term)

File: content/workers/runtime-apis/web-standards.md [hunk 1]
```diff
@@ -171,3 +171,22 @@ addEventListener('rejectionhandled', (event) => {
   console.log(event.reason);  // The value or Error with which the promise was rejected.
 });
 ```
+
+---
+
+## `navigator.sendBeacon(url[, data])`
+
+When the `[`global.navigator`](/workers/configuration/compatibility-dates/#global-navigator) compatibility flag is set, the [`navigator.sendBeacon(...)`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon) API is available to send an HTTP POST request containing a small amount of data to a web server. This API is intended as a means of transmitting analytics or simple diagnostics information asynchronously on a best-effort basis.
```
Comment (deadlypants1973) [inline content/workers/runtime-apis/web-standards.md]:
```suggestion
When the `[`global.navigator`](/workers/configuration/compatibility-dates/#global-navigator) compatibility flag is set, the [`navigator.sendBeacon(...)`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon) API is available to send an HTTP `POST` request containing a small amount of data to a web server. This API is intended as a means of transmitting analytics or simple diagnostics information asynchronously on a best-effort basis.
```

File: content/workers/runtime-apis/web-standards.md [hunk 2]
```diff
@@ -171,3 +171,22 @@ addEventListener('rejectionhandled', (event) => {
...
+
+For example, using `navigator.sendBeacon(...)`, you can replace:
```
Comment (deadlypants1973) [inline content/workers/runtime-apis/web-standards.md]:
```suggestion
For example, you can replace:
```

File: content/workers/runtime-apis/web-standards.md [hunk 3]
```diff
@@ -171,3 +171,22 @@ addEventListener('rejectionhandled', (event) => {
...
+
+```js
+const promise = fetch('https://example.com', { method: 'POST', body: 'hello world' });
+ctx.waitUntil(promise);
+```
+
+With
```
Comment (deadlypants1973) [inline content/workers/runtime-apis/web-standards.md]:
```suggestion
with `navigator.sendBeacon(...)`:
```

**Review by deadlypants1973 — APPROVED** [review]

**Comment by deadlypants1973** [comment]
Merging with James approval

**Pull request merged**